reference/5.1/Microsoft.PowerShell.Core/About/about_Automatic_Variables.md

```diff
@@ -188,10 +188,14 @@ blocks (which are unnamed functions).
   > You cannot use the `$input` variable inside both the Process block and the
   > End block in the same function or script block.
 
+Since `$input` is an enumerator, once you enumerate a value, it is no longer
+accessible in `$input`.
```

Comment:
I don't believe this is accurately represented.  Once you enumerate a value, `$input` itself is no longer accessible, not just the enumerated value. If you copy it into another object, such as in the example Sean gave, as a single operation, all it's properties are accessible in the new object.  However, if you access a single property in it, such as `$input.PropA`, `$input` is no longer accessible, not just `PropA`.

Reply:
I agree, my intention was to leave it open that if they did do something like what sean did in the example, implying that the value can be reused, however, $input would not be available. I'll see if I can clarify more.